README.md

```diff
@@ -16,8 +16,8 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
-
+```
+A constructor function is a generic object that contains keys that you will can use to define objects that you create in the future. To create a new object using a constructor, you need to use the keyword "new" to declare the new object that you are creating to be based off the constructor you have created. A prototype is typically a method that you want attached to a constructor, but by creating it as a prototype you only run the function when it is called.
```

Comment:
Yes, just to expand a constructor function is a function that returns a new instance/object :+1: